tests/mocks.ts

```diff
@@ -0,0 +1,2 @@
+export const allCats = ['Salem', 'Mimo', 'Tapi'];
+export const newCat = 'Millie';
```

Comment:
nit: would just move this to the test itself, theres no need for a whole file with two variables